src/foam/nanos/crunch/Capability.js

```diff
@@ -138,7 +143,7 @@ foam.CLASS({
         if ( ! this.getEnabled() ) return false;
 
         // check if permission is a capability string implied by this permission
-        if ( this.stringImplies(this.getId(), permission) ) return true;
+        if ( this.stringImplies(this.getName(), permission) ) return true;
```

Comment:
Why are we passing the capability 'name' to string implies?  Is the name not free form of the capability author's choosing?

Reply:
the name of a capability is checked in capability auth service to see if it implied the permission string. A permission string can either be the represented by the name or in the list of permissionsGranted of a capability